opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -26,8 +27,19 @@
 
 
 def auto_instrument():
+    package_to_exclude = Configuration().get("DISABLED_INSTRUMENTATIONS", [])
+    if isinstance(package_to_exclude, str):
+        package_to_exclude = package_to_exclude.split(",")
+        # to handle users entering "requests , flask" or "requests, flask" with spaces
+        package_to_exclude = [x.strip() for x in package_to_exclude]
```

Comment:
This line is added for any user entries containing spaces.  Having said that we do not have any control on the typo errors which users can make.